### PR TITLE
add license-header to dist

### PR DESCRIPTION
**Pull request by fcarstens**
Adds a license-header to a generated dist file. I'm not sure how we do best about the version information. Ideas? :smiley:

### CODE REVIEW

**Comment by jzaefferer** [comment]
In other projects the source header has a `@VERSION` placeholder, which during the build time is replaced with the `version` property from `package.json`. Maybe using https://www.npmjs.com/package/sed.js ? Or something like that.

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "js-reporters",
   "version": "1.0.0",
   "scripts": {
-    "build": "mkdirp dist/ && esperanto -b -i lib/index.js -t umd --name JsReporters | babel -o dist/js-reporters.js --blacklist strict -m umd",
+    "build": "mkdirp dist/ && esperanto -b -i lib/index.js -t umd --name JsReporters | babel --blacklist strict -m umd | cat lib/license-header.js - > dist/js-reporters.js",
```
Comment (leobalter) [inline package.json]: this is becoming too large.

I suggest splitting it to smaller steps or setting up a build file to be called here..

**Comment by jzaefferer** [comment]
It looks like we're going to have ES6 JS files in `test/` as well, so may as well create a separate build script.

**Comment by jzaefferer** [comment]
Still need to finish this.

**Comment by fcarstens** [comment]
Superseded by #26.

**Pull request closed without merging**